test/test-applications/integrations/Samples.Msmq/Program.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Linq;
+using System.Messaging;
+
+namespace Samples.Msmq
+{
+    public class Program
+    {
+        const string PrivateTransactionalQueuePath = ".\\Private$\\private-transactional-queue";
```

Comment:
I'm curious to see if that will work in the CI agent. That would be a pleasant surprise if the MSMQ service is activated.